vedtekter.md

```diff
@@ -1,63 +1,77 @@
 # Vedtekter for Offentlig PaaS
 
-Vedtekter for Offentlig PaaS. Vedtatt av stiftelsesmøtet 11.12.2024
+Vedtekter for Offentlig PaaS. Vedtatt av stiftelsesmøtet 11.12.2024, revidert på årsmøtet 2025.
 
 ## § 1 Formål
+
 Offentlig PaaS er en interesseorganisasjon for platformutviklere som jobber i offentlig sektor.
 
 Offentlig PaaS skal:
+
 * jobbe for å dele kunnskap om plattform og temaer under plattform.
 * fungere som et kontaktnettverk og samarbeidsforum for de som jobber med eller er interesserte i plattform.
 * være et samlingspunkt for plattformutviklere i offentlig sektor.
 
-
 ## § 2 Organisatorisk tilknytning
-Alle som er ansatt i offentlig sektor, eller selskaper hvor offentligheten er majoritetseier, kan bli medlem. Medlemskap tildeles automatisk ved registrering i Offentlig PaaS Slacken.
 
-## § 3 Medlemmer
-Medlemmer i foreningen er personer som har meldt seg til å arrangere faglige arrangementer i Offentlig PaaS.
+Alle som er ansatt i offentlig sektor, eller selskaper hvor offentligheten er majoritetseier, kan bli medlem.
+
+## § 3 Medlemmskap
+
+Medlemskap tildeles automatisk ved registrering i Offentlig PaaS Slacken.
 
 ## § 4 Stemmerett og valgbarhet
-Alle medlemmer har stemmerett og er valgbare til tillitsverv i foreningen.
+
+Alle registrerte organisasjoner har én stemme hver, og alle registrerte medlemmer er valgbare til tillitsverv i Offentlig PaaS.
```

Comment:
Det at "registrert organisasjon" innføres som begrep ved stemmeavgivning uten å være definert gjør dette litt uklart.

Reply:
Ja, jeg begynte på en litt lenger utgreiing, men ble ikke helt fornøyd. Får gjøre et nytt forsøk for jeg er enig at det er uheldig. Evnt. kan "registrerte organisasjoner" byttes ut med "representerte organisasjoner".

Reply:
Forslag til ordlyd:

"Alle registrerte medlemmer er valgbare til tillitsverv i Offentlig PaaS, og alle organisasjoner representert av medlemmene tildeles én stemme hver".

Reply:
Det er mye tydeligere 🙌